src/components/credit-card-date/code/views/creditCardDateView.jsx

```diff
@@ -0,0 +1,78 @@
+var React = require('react');
+var Payment = require('payment');
+
+module.exports = React.createClass({
+
+  intent: null,
+  formatting: false,
+
+  propTypes: {
+    label: React.PropTypes.string,
+    expireDate: React.PropTypes.string,
+    defaultValue: React.PropTypes.string,
+    name: React.PropTypes.string,
+    id: React.PropTypes.string,
+    valid: React.PropTypes.bool,
+    errorMessage: React.PropTypes.string
+  },
+
+  getInitialState: function() {
+    return {
+      value: this.props.defaultValue || '',
+      error: null,
+      valid: true
+    };
+  },
+
+  getDefaultProps: function() {
+    return {
+      placeHolder: 'MM / YY',
+      name: 'credit-card-date',
+      id: 'credit-card-date',
+      errorMessage: 'Invalid Date'
+    };
+  },
+
+  handleChange: function(e){
+
+    var elm = document.querySelector('.credit-card-date-input');
```

Comment:
This doesn't work if we have multiple card date inputs on the page such as "start date" and "expiry date". The second input would never be formatted. Does e not provide us the active element?